.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+orbs:
+  onegini-build: onegini/onegini-build@2
+
+shared: &shared
+  component_type: library
+  context: dev-context
+  executor:
+    name: onegini-build/maven-builder
+    tag: "11.0"
+
+filter_all_branches_and_tags: &filter_all_branches_and_tags
```

Comment:
Little bit overkill since this is only used once ;)

Reply:
It's consistent with other projects and it will allow us to quickly add other steps. Do you want me to change it?

Reply:
It's gone now